src/core/models/quarters.ts

```diff
@@ -0,0 +1,51 @@
+import { Schema, model } from "mongoose";
+
+const QuarterSchema = new Schema({
+    
+    currentYear: { 
```

Comment:
Should just be `year` since its value doesn't change once it has been created

Reply:
i thought the telegram bot will always ask us to input the year and roi anytime we want to make an entry?

Reply:
Yes it will, but a particular record will not be updated after it has been created so it shouldn't be named `currentYear` as if we are going to change that record later